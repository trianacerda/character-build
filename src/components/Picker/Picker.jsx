import React from 'react';
import { useState } from 'react';

function Picker({ head }) {
  const [head, setHead] = useState('');
  const [middle, setMiddle] = useState('');

  const headPictures = [];
  const middleOptions = [];
  return (
    <>
      <label>
        Head Options
        <select value={head} onChange={(e) => onHeadChange(e.target.value)}>
          {headPictures.map((picture) => (
            <option key={picture}>{picture}</option>
          ))}
        </select>
      </label>

      <label>
        Middle Options
        <select value={middle} onChange={(e) => onHeadChange(e.target.value)}>
          {middleOptions.map((option) => (
            <option key={option}>{option}</option>
          ))}
        </select>
      </label>
    </>
  );
}

export default Picker;
