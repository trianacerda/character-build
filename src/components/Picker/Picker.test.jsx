import { screen, render } from '@testing-library/react';
import Picker from './Picker';

it('should render Picker element on page', () => {
  const { container } = render(<Picker />);
  const element = screen.getByLabelText(/Middle Options/);

  expect(container).toMatchSnapshot();
  expect(element).toBeInTheDocument();
});
