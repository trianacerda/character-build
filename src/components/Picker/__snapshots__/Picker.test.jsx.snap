// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Picker element on page 1`] = `
<div>
  <label>
    Head Options
    <select>
      <option>
        rip
      </option>
      <option>
        beth
      </option>
      <option>
        john
      </option>
      <option>
        triana
      </option>
    </select>
  </label>
  <label>
    Middle Options
    <select>
      <option>
        cowboy
      </option>
      <option>
        code
      </option>
      <option>
        hoodie
      </option>
      <option>
        kobe
      </option>
    </select>
  </label>
  <label>
    Bottom Options
    <select>
      <option>
        cowboy
      </option>
      <option>
        rainbow
      </option>
      <option>
        mc
      </option>
      <option>
        gummybear
      </option>
    </select>
  </label>
  <label>
    Catch Phrase:
    <input
      type="text"
      value=""
    />
  </label>
  <button
    type="button"
  >
    Add your catchphrase!
  </button>
</div>
`;
